sigma/backends/loki/loki.py

```diff
@@ -271,6 +273,9 @@ class LogQLBackend(TextQueryBackend):
     compare_operators: ClassVar[Dict[SigmaCompareExpression.CompareOperators, str]]
     case_sensitive_match_expression: ClassVar[str]
 
+    # exists modifier is not supported in LogQL (yet?)
+    field_exists_expression: ClassVar[str] = ""

```

Comment:
This disables the `exists` check. I asked the Loki team internally for comments.

Reply:
Implemented.